docs/program-setup/serial/serial_communication.md

```diff
@@ -6,86 +6,100 @@ The Arm Mbed microcontroller on your board can communicate with a host PC over t
+- Print out messages to a PC terminal (useful for debugging).
+- Read input from a PC keyboard.
+- Communicate with applications running on a PC to exchange data.
 
-## Hello, world
+## Hello World - printing messages
 
-This program prints a "Hello World" message that you can view on a [terminal application](#using-terminal-applications). Communication over the USB serial port uses the standard serial interface. Specify the internal (USBTX, USBRX) pins to connect to the serial port routed over USB:
+This program prints a "Hello World!" message that you can view on a serial terminal. Mbed OS redirects any `printf()` statements to the board's debug USB serial.
 
-[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/Tutorials_SerialComm/Serial_HelloWorld/)](https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/Tutorials_SerialComm/Serial_HelloWorld/main.cpp)
+[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/Tutorials_SerialComm/Serial_STDOUT/)](https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/Tutorials_SerialComm/Serial_STDOUT/main.cpp)
+
+## Viewing output in a serial terminal
+
+Serial terminals run on your host PC. They provide an interface where your Mbed board can print and where you can type characters back to your board.
+
+Mbed CLI provides a serial terminal that is configured with a default baud rate of `9600`. When a single board is connected, run:
 
-## Using terminal applications
+```
+mbed sterm
+```
 
-Terminal applications run on your host PC. They provide a window where your Mbed board can print and where you can type characters back to your board.
+(If you have multiple boards connected, please refer to [Additional options of mbed sterm](#additional-options-of-mbed-sterm).)
 
-<span class="tips">**Serial configuration:** The standard setup for the USB serial port is 9600 baud, 8 bits, 1 stop bit, no parity (9600-8-N-1)</span>
+`mbed sterm` starts message printing.
 
-### Installing an application for Windows
+Restart the application using the board's reset button, or by entering <kbd>Ctrl + B</kbd> in the serial terminal.
+The example prints "Hello World!" to the terminal after the reset.
 
-There are many terminal applications for Windows, including:
+To close the serial terminal, enter <kbd>Ctrl + C</kbd>.
 
-- [CoolTerm](http://freeware.the-meiers.org/) - this is the application we use in this example. We use it often because it usually "just works".
-- [Tera Term](http://sourceforge.jp/projects/ttssh2/files).
-- [PuTTY](http://www.chiark.greenend.org.uk/~sgtatham/putty/).
-- Some Windows PCs come with **Hyperterminal** installed.
+## Additional options of mbed sterm
 
-### Configuring the connection
+- If you have multiple boards connected:
+    1. Run `mbedls` to find the port of the board you want to use.
```

Comment:
I cannot find any page for `mbedls` on Mbed OS's documentation site. The info in the link seems outdated as it says "Development moved", and the steps to install this command don't apply anymore since `mbedls` is now included in Mbed CLI suite.